dashboard/app/controllers/sessions_controller.rb

```diff
@@ -6,6 +6,12 @@ class SessionsController < Devise::SessionsController
   def new
     session[:return_to] ||= params[:return_to]
     @already_hoc_registered = params[:already_hoc_registered]
+    @hide_oauth_sidebar = false
+    if params[:providerNotLinked]
+      flash.now[:alert] = I18n.t 'auth.not_linked', provider: I18n.t("auth.#{params[:providerNotLinked]}")
```

Comment:
You are trying to i18n the name of the provider? Presumably this is unintentional?